webpack/assets/javascripts/react_app/components/common/CardTemplate/CardItem.js

```diff
@@ -0,0 +1,102 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import Skeleton from 'react-loading-skeleton';
+import {
+  DataList,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  DataListCell,
+  Card,
+  CardTitle,
+  CardBody,
+  Accordion,
+  AccordionItem,
+  AccordionContent,
+  AccordionToggle,
+  GridItem,
+} from '@patternfly/react-core';
+
+const CardItem = ({ content, header, children }) => {
+  const [activeAccordion, setActiveAccordion] = useState(0);
+
+  const onToggle = id => {
+    if (id === activeAccordion) {
+      setActiveAccordion('');
+    } else {
+      setActiveAccordion(id);
+    }
+  };
+
+  return (
+    <GridItem xl2={3} md={6} lg={5}>
+      <Card isHoverable>
+        <CardTitle>{header}</CardTitle>
+        <CardBody>
+          {children || (
```

Comment:
This enables us to add custom content from plugins.
You can find a usage example here https://github.com/theforeman/foreman_remote_execution/pull/549